src/EscapeRoomServiceProvider.php

```diff
@@ -53,6 +53,9 @@ public function bootingPackage()
     {
         parent::bootingPackage();
 
+        // Make sure we dont accidentally steal any nova API or company routes
+        Route::pattern('market', '^(?!(nova-api|nova-vendor|company))[^\/]+$');
```

Comment:
@wolfrednicolas this will fix the issue w/cards and filters returning 404.  This pattern definition exists in the tipoff/locations package as well, but the tipoff/escape-room package is being booted first, so it needs it as well.

Reply:
thanks it works!!

Reply:
This is great! Thank you both!